src/pages/Home.js

```diff
@@ -26,6 +30,7 @@ const Home = () => {
     e.preventDefault();
     if (existingTokens.includes(userToken) && userToken !== '') {
       saveToken(userToken);
+      setHasToken(userToken);
       navigate('/list');
     } else {
```

Comment:
one thing I see here is there is some duplication. It seems everytime setHasToken is called, the user is also navigatied to the list. After leaving this for a few days, do you see a way that this could be finessed a little?

Reply:
Ah yeah - consolidating the two into one function?